tests/target/issue-1021.rs

```diff
@@ -15,7 +15,7 @@ fn main() {
         (true, ..) => (),
         (.., true) => (),
         (..) => (),
-        (_) => (),
+        (_,) => (),
```

Comment:
The current test is actually incorrect: `(_,)` and `(_)` are different.

Reply:
The change from `(_,)` to `(_)` in this file was introduced in a recent PR, so I believe rustfmt had been working correctly before that.